java/opendcs/src/main/java/lrgs/db/LrgsDatabase.java

```diff
@@ -321,7 +322,10 @@ public void commit()
 			conn.commit();
 			conn.clearWarnings();
 		}
-		catch(SQLException ex) {}
+		catch(SQLException ex)
+		{
+			throw new LrgsDatabaseException("Unable to commit changes.", ex);
```

Comment:
behavior change?

Reply:
yeah, but it seemed appropriate, that's quite the silent error. Could just change it to logging though.